app/presenters/job_presenter.rb

```diff
@@ -26,6 +26,17 @@ def list(jobs)
         single(job)
       end.compact
     end
+    
+    def list_by_cluster_id(jobs, cluster_id)
+      jobs.map do |job|
+        unless job['payload'].blank?
```

Comment:
Prefer guard clauses over increasing indentation.
How about:
`next if job['payload'].blank?`

Reply:
Shouldnt it be 

`next if job['payload'].present?`

Reply:
`next` means to proceed immediately to the next iteration of the loop.
So, we proceed immediately to processing the next job, if this job's payload is blank.

Reply:
Gotcha, fixed, please check